examples/module1/lesson1/validate-it/index.js

```diff
@@ -1,34 +1,36 @@
-function validator() {
-  const input = document.getElementById('input');
-  const button = document.getElementById('button');
-  const button2 = document.getElementById('button2');
+// CONSTANTS
+const VALIDATE_SUCCESS_MESSAGE = 'Valid';
```

Comment:
Komentarz typu 'sztuka dla sztuki', ale stałe zapisane upper-snake-case powinny być zarezerwowane dla stałych konfiguracyjnych. Tutaj, z racji tego, że oba te komunikaty nie są zwracane z jakiegoś API (czyli można założyć, że nie będą używane jako złoty standard), lepiej je zadeklarować za pomocą `camelCase`:

```
// Validation Messages
const validateSuccessMessage = 'Valid';
const validateErrorMessage = 'Invalid';
```

ale wiadomo, to są tylko standardy i upodobania ;p

Reply:
Poprawiłem kod zgodnie z powyższą sugestią - ma to sens ;)